docs/openapi-access.md

```diff
@@ -0,0 +1,170 @@
+```
+
+You can easily check schema version at runtime:
+```bash
+curl -s http://localhost:8000/api/eda/v1/openapi.json | jq -r .info.version
+```
+
+#### In a product
+If you are running EDA within an AAP instance and not a standalone deployment of EDA, replace `localhost:8000` with your controller URL (or base URL behind any gateway). The paths stay the same. For example:
+- `https://your-host/api/eda/v1/openapi.json`
+- `https://your-host/api/eda/v1/docs/`
+
+In OpenShift:
+
+- `https://example-aap.apps.my-cluster-name.openshift.org/api/eda/v1/openapi.json`
+
+**Notes**:
+- Endpoints require authentication by default. See [Authentication](#authentication) below.
+- For TLS with self-signed certs, use `-k`/`--insecure` for local testing tools (curl, HTTP clients, etc).
+
+---
+
+### Configuration notes
+API prefix can be customized (default: `api/eda`). If changed, schema endpoints move accordingly.
+The schema generator is configured via DRF Spectacular; relevant settings include:
+- Title, `info.version`, path prefix trimming, and servers URL.
+- Anonymous access can be enabled by adjusting REST Framework permissions if needed (although, not recommended for production).
+
+---
+
+### Authentication
+By default, schema endpoints require authentication (standard API auth with RBAC).
+
+Common ways to authenticate are:
+- Basic auth (user:password)
+- Session cookie (after logging in via the UI or session endpoint)
+- Bearer token/JWT (when issued by your identity provider or access gateway)
+
+Examples:
+- Basic auth:
+  ```bash
+  curl -u admin:password \
+    "http://localhost:8000/api/eda/v1/openapi.json" -o eda.json
+  ```
+- Bearer token:
+  ```bash
+  curl -H "Authorization: Bearer $TOKEN" \
+    "https://localhost:8000/api/eda/v1/openapi.json" -o eda.json
+  ```
+
+---
+
+### Versioning and update cadence
+The OpenAPI `info.version` is set from the installed `aap-eda` package version.
+Runtime schemas are generated dynamically from the running service; they reflect whatever version is deployed.
+For upstream usage, there is no central public catalog of static EDA specs. If you need a stable reference:
+- Export the spec from a known EDA version (e.g., the image or package version you deploy).
+- Check the exported file into your own repository under a versioned path, for example:
+  - `openapi/eda-<version>.json` or `openapi/eda-<version>.yaml`
+- Use that pinned file for SDK generation, CI, and documentation.
+
+Typical workflow when upgrading:
+- Upgrade your EDA environment, export the new spec, and commit it
+- Diff the new spec against your previously pinned version to identify changes
+- Regenerate clients and run integration tests against the new spec
+
+**Notes**:
+- If you generate client code, pin to a specific spec version for reproducible builds.
+- For integration testing, compare the live spec to your pinned spec snapshot to catch breaking changes early.
+
+---
+
+### Code examples
```

Comment:
Did we vet these examples? Just want to make sure we know they work for sure before putting them in the docs